src/main/io/gps.c

```diff
@@ -1578,9 +2114,46 @@ static bool UBLOX_parse_gps(void)
     uint32_t i;
 
     *gpsPacketLogChar = LOG_IGNORED;
+#define CLSMSG(cls, msg) (((cls) << 8) | (msg))
+    switch (CLSMSG(_class, _msg_id)) {
+    case CLSMSG(CLASS_MON, MSG_MON_VER): {
+        char *token = strtok(_buffer.ver.swVersion, " ");
+        while (token != NULL) {
+            float f = fastA2F(token);
```

Comment:
this is quite wasteful

Reply:
How could it best be coded?

Reply:
I'll get to it soon

Reply:
@ledvinap should I close this comment?  Otherwise please share your code suggestion.